src/components/calcite-combobox-item/calcite-combobox-item.tsx

```diff
@@ -0,0 +1,169 @@
+import {
+  Component,
+  Element,
+  Event,
+  EventEmitter,
+  Host,
+  Method,
+  Prop,
+  h,
+  State,
+  Watch
+} from "@stencil/core";
+
+import { CSS } from "./resources";
+
+@Component({
+  tag: "calcite-combobox-item",
+  styleUrl: "./calcite-combobox-item.scss",
+  shadow: true
+})
+export class CalciteComboboxItem {
+  // --------------------------------------------------------------------------
+  //
+  //  Properties
+  //
+  // --------------------------------------------------------------------------
+
+  /**
+   * When true, the item cannot be clicked and is visually muted.
+   */
+  @Prop({ reflect: true }) disabled? = false;
```

Comment:
Do we need disabled items? If so, should key navigation skip over them?

Reply:
Yeah we should skip over them if disabled. Do we need them though?